project/src/main/webapp/script.js

```diff
@@ -136,6 +136,36 @@ function createTrendElement(trend, val) {
   return trendElement;
 }
 
+/** Displays trend frequency bubbles on page. */
+function getTrendBubbles(content) {
+  const bubbleContainer = document.getElementById('trend-frequency-data');
+  const bubbleSizes = getTrendBubbleSize(content);
+  const keys = bubbleSizes.keys();
+  let key = keys.next().value;
+  while (key != undefined) {
+    bubbleContainer.appendChild(createTrendBubbles(key,bubbleSizes.get(key)));
+    key = keys.next().value;
+  }
+}
+
+/** Creates an element that represents a trend and its frequency. */
+function createTrendBubbles(trend, size) {
+  const bubbleElement = document.createElement('div');
+  bubbleElement.className = 'bubbles';
+  let style = 'width:' + size + 'vw; height:' + size + 'vw; line-height:' + size + 'vw; font-size:' + size/10 + 'vw;';
```

Comment:
Can we have a nicer style builder instead of appending size and some magic conts/strings together?